### PR TITLE
Created conservation tests for compressible NS element

### PR DESCRIPTION
**Pull request by EduardGomezEscandell**
**📝 Description**
This PR introduces new unit tests to the element (and future conditions) by checking that some conservation conditions are fulfilled. The goal is to make implementation of new features easier. The tests are detailed here:

**Test 1: RIGID BODY TRANSLATION** ✔️
 - ∇ρ = 0
 - ∇(ρu) = 0
 - ∇ e_total = 0

The analytical solution is:
- ∂U/∂t = 0

**Test 2: UNIFORM ENERGY** ✔️
 - ∇ e_total = 0
 - V = 0
 - λ = 0

The analytical solution is:
- ∂U/∂t = 0

**Test 3: RIGID BODY ROTATION** ✔️
 - ∇ρ = 0
 - λ = 0
 - V = ω[-y, x]^T  .................   (Pure rotation around origin, ω is angular frequency)
 - p = ½ρω²r²+p0 .................     (=centripetal force + pressure at origin)

The analytical solution is:
- ∂U/∂t = 0

**🆕 Changelog**
- Created aformentioned unit tests

### CODE REVIEW

**Comment by EduardGomezEscandell** [comment]
@rubenzorrilla

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_compressible_navier_stokes_explicit_conservation.cpp [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+/**
+ * @brief Test the 2D explicit compressible Navier-Stokes element and condition RHS
+ * This is a conservation test.
+ *
+ * RIGID BODY TRANSLATION
+ * N-S equations say that with:
+ *  - ∇U = 0
+ * then the time derivatives should be zero.
+ */
+KRATOS_TEST_CASE_IN_SUITE(CompressibleNavierStokesExplicit2D_ConservationRigidTranslation, FluidDynamicsApplicationFastSuite)
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/test_compressible_navier_stokes_explicit_conservation.cpp]:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(CompressibleNavierStokesExplicit2DConservationRigidTranslation, FluidDynamicsApplicationFastSuite)
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_compressible_navier_stokes_explicit_conservation.cpp [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+ * @brief Test the 2D explicit compressible Navier-Stokes element and condition RHS
+ * This is a conservation test.
+ * 
+ * N-S equations say that given the following conditions:
+ *  - V = 0
+ *  - ∇ e_total = 0
+ *  - λ = 0
+ * the time derivatives should be zero
+ */
+KRATOS_TEST_CASE_IN_SUITE(CompressibleNavierStokesExplicit2D_ConservationStatic, FluidDynamicsApplicationFastSuite)
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/test_compressible_navier_stokes_explicit_conservation.cpp]:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(CompressibleNavierStokesExplicit2DConservationStatic, FluidDynamicsApplicationFastSuite)
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_compressible_navier_stokes_explicit_conservation.cpp [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+ *              3
+ *             /|\
+ *            / | \
+ *     ------/--+--\-----------
+ *          /   |   \
+ *         1 ------- 2
+ *              |
+ * 
+ */
+KRATOS_TEST_CASE_IN_SUITE(CompressibleNavierStokesExplicit2D_ConservationRigidRotation, FluidDynamicsApplicationFastSuite)
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/test_compressible_navier_stokes_explicit_conservation.cpp]:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(CompressibleNavierStokesExplicit2DConservationRigidRotation, FluidDynamicsApplicationFastSuite)
```

**Review by rubenzorrilla — COMMENTED** [review]
Minor suggestions on the naming.

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**